enterprise/server/cmd/cache_proxy/cache_proxy.go

```diff
@@ -259,5 +260,12 @@ func registerInternalGRPCServices(grpcServer *grpc.Server, env *real_environment
 		return status.InternalErrorf("CacheProxy: error starting local contentaddressablestorage server: %s", err.Error())
 	}
 	repb.RegisterContentAddressableStorageServer(grpcServer, localCAS)
+
+	localAC, err := action_cache_server.NewActionCacheServer(env)
+	if err != nil {
+		return status.InternalErrorf("CacheProxy: error starting local actioncache server: %s", err.Error())
+	}
+	repb.RegisterActionCacheServer(grpcServer, localAC)
+	env.SetLocalActionCacheServer(localAC)
```

Comment:
is there a reason not to register the client that calls to the grpc server instead? the equivalent registrations for the bs and cas clients are up at about line 180:
```
	conn, err := grpc_client.DialInternal(env, fmt.Sprintf("grpc://localhost:%d", grpc_server.InternalGRPCPort()))
	if err != nil {
		return status.InternalErrorf("CacheProxy: error starting local bytestream gRPC server: %s", err.Error())
	}
	env.SetLocalByteStreamClient(bspb.NewByteStreamClient(conn))
	env.SetLocalCASClient(repb.NewContentAddressableStorageClient(conn))
```